perfdash/www/script.js

```diff
@@ -21,7 +21,10 @@ var PerfDashApp = function(http, scope) {
     this.scope = scope;
     this.selectedResource = "pods";
     this.selectedVerb = "GET";
+    this.testNames = [];
+    this.testName = "Density";
```

Comment:
You can initialize it with null and get first available value after line 39. With `this.testNames[0]`.